From 8910f15f2e71b65f527af2b26d2fd15df5086a9f Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 6 May 2020 10:07:04 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable
 CONFIG_DM_ANDROID_VERITY_AT_MOST_ONCE_DEFAULT_ENABLED

According to the Android 10 Go edition Device Configuration Guide 1.0.

Change-Id: Ibd9adb2ad2a000f0332bca6524bf427710572459
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 72a812d33346..34cb0d207dd3 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -282,6 +282,7 @@ CONFIG_DM_VERITY=y
 CONFIG_DM_VERITY_AVB=y
 CONFIG_DM_VERITY_FEC=y
 CONFIG_DM_BOW=y
+CONFIG_DM_ANDROID_VERITY_AT_MOST_ONCE_DEFAULT_ENABLED=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
 # CONFIG_NET_VENDOR_3COM is not set
-- 
2.35.3

